gui/journal.lua

```diff
@@ -761,6 +819,8 @@ function JournalScreen:onDismiss()
     view = nil
 end
 
+view = nil
```

Comment:
this resets `view` on every script invocation or module reload. if you want a top-level declaration for clarity's sake, use `view = view or nil` (but it's not required, and you can just delete this line entirely)